docs/nightly/en/getting-started/installation/greptimedb-standalone.md

```diff
@@ -23,6 +23,16 @@ You can run GreptimeDB in the standalone mode:
 ./greptime standalone start
 ```
 
+:::tip NOTE
+GreptimeDB binds to `127.0.0.1` by default. If you need to accept connections from other addresses, you can start with the following parameters.
```

Comment:
Do we need to add some security notes? For example, do not expose the database to public networks.